jormungandr/src/fragment/process.rs

```diff
@@ -84,6 +83,15 @@ impl Process {
                 .map_err(Error::PersistentLog)
         }
 
+        let min_logs_size = n_pools * self.pool_max_entries;
+        if self.logs_max_entries < min_logs_size {
+            tracing::warn!(
+                "Having 'log_max_entries' < 'pool_max_entries' * n_pools is not recommendend"
+            );
+            tracing::warn!("Overriding 'log_max_entries' to {}", min_logs_size);
+        }
```

Comment:
Should be a single log entry IMO.

Reply:
I was unsure about the best way to do this. I split that into two because otherwise it would be quite a long line, but I understand also why it should go together